docs/demo-slack-research-agent.md

```diff
@@ -0,0 +1,172 @@
+
+> **Prerequisites:**  
+> Ensure you've completed the Kagenti platform setup as described in the [Installation](./demos.md#installation) section. This demo uses `SLACK_BOT_TOKEN` so please include this. 
+
+> **Note:**
+> This demo has been tested skipping the install of mcp-gateway:
+> `uv run kagenti-installer --skip-install mcp_gateway`
+
+You should also open the Agent Platform Demo Dashboard as instructed in the [Connect to the Kagenti UI](./demos.md#connect-to-the-kagenti-ui) section.
+
+---
+
+## Import New Agent
+
+To deploy the Weather Agent:
+
+1. Navigate to [Import New Agent](http://kagenti-ui.localtest.me:8080/Import_New_Agent#import-new-agent) in the Kagenti UI.
+2. In the **Select Namespace to Deploy Agent** drop-down, choose the `<namespace>` where you'd like to deploy the agent. (These namespaces are defined in your `.env` file.)
+3. Under [**Select Environment Variable Sets**](http://kagenti-ui.localtest.me:8080/Import_New_Agent#select-environment-variable-sets), select:
+   - `mcp-slack`
+   - `ollama`
+   - `slack-researcher-config`
+Note that slack-researcher-config defines `TASK_MODEL_ID` as the model that is used by the agent. You may use `openai` instead of `ollama`, but will need to specify a different `TASK_MODEL_ID`. This demo has been tested with `openai` environment with `TASK_MODEL_ID=gpt-4o-mini-2024-07-18`
+4. In the **Agent Source Repository URL** field, use the default:
+   <https://github.com/kagenti/agent-examples>
+   Or use a custom repository accessible using the GitHub ID specified in your `.env` file.
+5. For **Git Branch or Tag**, use the default `main` branch (or select another as needed).
+6. Set **Protocol** to `a2a`.
+7. Under [**Specify Source Subfolder**](http://kagenti-ui.localtest.me:8080/Import_New_Agent#specify-source-subfolder):
+   - Click `Select from examples`
+   - Choose: `a2a/slack_researcher`
+8. Click **Build New Agent** to deploy.
+
+---
+
+## Import New Tool
+
+To deploy the slack Tool:
+
+1. Navigate to [Import New Tool](http://kagenti-ui.localtest.me:8080/Import_New_Tool#import-new-tool) in the UI.
+2. Select the same `<namespace>` as used for the agent.
+3. In the **Select Environment Variable Sets** section, select:
+   - `mcp-slack-config`
+4. Use the same source repository:
+   <https://github.com/kagenti/agent-examples>
+5. Choose the `main` branch or your preferred branch.
+6. Set **Select Protocol** to `MCP`.
+7. Under **Specify Source Subfolder**:
+   - Select: `mcp/slack_tool`
+8. Click **Build New Tool** to deploy.
+
+---
+
+## Validate The Deployment
+
+Depending on your hosting environment, it may take some time for the agent and tool deployments to complete.
+
+To verify that both the agent and tool are running:
+
+1. Open a terminal and connect to your Kubernetes cluster.
+2. Use the namespace you selected during deployment to check the status of the pods:
+
+   ```console
+   installer$ kubectl get po -n <namespace>
```

Comment:
```suggestion
   installer$ kubectl get pods -n <namespace>
```

It is better to spell out commands in docs, for users new to Kubernetes, even if you would never type the full word.